test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5067,15 +5066,14 @@ BOOST_AUTO_TEST_CASE(inline_assembly_storage_in_modifiers)
 			modifier m {
 				assembly {
 					x := 2
-					pop
 				}
 				_;
 			}
 			function f() m {
 			}
 		}
 	)";
-	CHECK_ERROR(text, DeclarationError, "");
+	CHECK_ERROR(text, DeclarationError, "Variable not found or variable not lvalue.");
```

Comment:
* `{ let y := 3 { 2 { let x := y } pop} pop }` should make the analyzer unhappy because the stack heights do not match at the end of the outer block
* `{ let y := 3 { 2 { let x := y } pop} pop 2}` should make the analyzer unhappy because too-deep stack elements are disturbed.